content/2. Ondersteunende-informatie/3. Processen/3. Ontwerpproces/1. Opstellen ontwerp/Schematechnieken/1. Klassendiagram.md

```diff
@@ -0,0 +1,163 @@
+    naam
+    email
+}
+
+
+class Persoon2 {
+<< abstract >>
+	naam
+    email
+}
+
+class Persoon3 {
+<< interface >>
+	naam
+    email
+}
+
+```
+*Diagram 3: voorbeelden van veelgebruikte typen klassen*
+#### Voorbeelden van relaties
+Een applicatie is opgebouwd uit verschillende klassen. Door middel van associates geef je de samenhang tussen klassen weer. Er zijn verschillende soorten associaties en in onderstaand diagram vind je de meest gebruikte associaties.
+``` mermaid
+
+classDiagram
+classI -- classJ : Link
+Hotel --> Kamer : Directieve association
+Dier <|-- Hond : Overerving
+Persoon ..> Boek : Afhankelijkheid
+
+```
+*Diagram 4: Voorbeelden van relaties*
+
+| Type                   | Toelichting                                                                                                                                                                                                                                                                                                                                                   |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Link                   | De standaard associatie geeft een algemene relatie weer tussen de twee klassen.                                                                                                                                                                                                                                                                               |
+| Directieve association | Dit type relatie geeft richting aan tussen de klassen. Afhankelijk van de pijl kun je één of beide richtingen op navigeren.<br>In *Diagram 4* kun je bijvoorbeeld wel navigeren van `Hotel` naar `Kamer` maar bij een `Kamer` kun je niet direct naar het gerelateerde `Hotel` navigeren.                                                                     |
+| Overerving             | Door de overervingsrelatie te gebruiken geef je aan dat een klasse uitbreiding is van een andere klasse.<br>In *Diagram 4* zie je bijvoorbeeld dat een `Hond` een overerving is van `Dier`. Een `Kat` zou dat net zo goed kunnen zijn.                                                                                                                        |
+| Afhankelijkheid        | Een afhankelijkheid kun je gebruiken om relaties weer te geven die niet blijken uit de Attributen. <br>In Diagram 4 zie je dat een persoon afhankelijk is van een `Boek`. Dit is het geval omdat een `Persoon` in inhoud van een `Boek` moet kunnen lezen. Er ligt dan niet perse een directe relatie tussen `Persoon` en `Boek` maar we een afhankelijkheid. |
+
+### Het proces om het diagram te maken
```

Comment:
Dit is een kopje wat in Procedurele informatie moet staan. Hierbij moet alleen nog het format van de template worden toegepast